app/sentinel/logger/logger_port.go

```diff
@@ -1,3 +1,11 @@
 package logger
 
-const LOGGER_PORT = ":50051"
+import "os"
+
+func SentinelLoggerUrl() string {
+	u := os.Getenv("SENTINEL_LOGGER_URL")
+	if u == "" {
+		return "[::]:50051"
```

Comment:
Binding to `[::]` can be a security issue;

`localhost:50051` would have been better.

I’ll create a separate issue for that.